internal/storage/disk.go

```diff
@@ -0,0 +1,68 @@
+package storage
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/demurk/tinyurl/internal/config"
+)
+
+var lastUUID = 0
+
+type urlData struct {
+	UUID        int    `json:"uuid"`
+	ShortURL    string `json:"short_url"`
+	OriginalURL string `json:"original_url"`
+}
+
+func RestoreURLsFromFile() {
```

Comment:
```suggestion
func RestoreURLsFromFile() error {
```
Ошибку всегда необходимо тащить из нижних функций на самый верх и уже там принимать решение - что с ней делать.